aptos-move/framework/aptos-framework/sources/configs/gas_schedule.spec.move

```diff
@@ -39,7 +41,7 @@ spec aptos_framework::gas_schedule {
         use aptos_framework::aptos_coin::AptosCoin;
         use aptos_framework::transaction_fee;
 
-        pragma timeout = 60;
+        pragma timeout = 100;
```

Comment:
Why did this change?

Reply:
The test was timeout in my laptop. I forgot to change it back. Fixed.